frog/imports/api/activities.js

```diff
@@ -99,6 +150,15 @@ Meteor.methods({
     Activities.remove({})
   },
 
+  'graph.flush.operator': (operatorId) => {
+    Operators.remove({ _id: operatorId })
+  },
+
+  'graph.flush.operators': (activityId) => {
+    Operators.remove({ 'from._id': activityId })
+    Operators.remove({ 'to._id': activityId })
```

Comment:
We only copied the plane and the _id field of the source and target activities, as an activity position, duration and other properties may change. In that case we would have to always modify the operators which is tedious. Thus, we copied only the plane and the id as both were used a lot. 
In Graph.jsx :
```
addNewOperator = (target) => {
    if (this.state.currentSource != null && target != this.state.currentSource) {
      this.setState({currentSource:null, loaded:false});
      const fromAc = {plane: this.state.currentSource.plane, _id: this.state.currentSource._id}
      const toAc = {plane: target.plane, _id: target._id}
      addGraphOperator({_id: uuid(), graphId: this.props.graphId, from: fromAc, to:toAc})
    }
  }
```